ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/restore/view/RestoreView.kt

```diff
@@ -321,11 +340,23 @@ private fun Autocomplete(
 
         LazyRow(highlightModifier.testTag(RestoreTag.AUTOCOMPLETE_LAYOUT)) {
             items(it) {
-                Button(
-                    modifier = Modifier.testTag(RestoreTag.AUTOCOMPLETE_ITEM),
-                    onClick = { onSuggestionSelected(it) }
+                Surface( // TODO create chip
+                    modifier = modifier
+                        .padding(4.dp)
+                        .testTag(RestoreTag.AUTOCOMPLETE_ITEM)
+                        .clickable { onSuggestionSelected(it) },
+                    shape = RoundedCornerShape(8.dp),
+                    color = MaterialTheme.colorScheme.secondary,
+                    shadowElevation = 8.dp
                 ) {
-                    Text(it)
+                    Text(
+                        text = it,
+                        style = MaterialTheme.typography.bodyLarge,
+                        color = MaterialTheme.colorScheme.onSecondary,
+                        modifier = Modifier
+                            .padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp)
```

Comment:
We're starting to introduce a lot of padding values.

We should start to think about how we might account for these in a larger design system.  I don't think we have that architecture figured out just yet, but I want to note it because we'll probably need to revisit these in the near future.

@HonzaR do you have any further thoughts since you've been thinking about this?

Reply:
I 100% agree, we could take it a step. further and create our own padding Modifiers . what do you think?

Reply:
Yes, could be. Good idea. My plan is to prepare a separate Dimens class, which will hold needed spacing values, and also enables us to switch between these predefined group of spacing for small/normal/large screens as well as for portrait/landscape, etc.